insets/src/main/java/com/google/accompanist/insets/Insets.kt

```diff
@@ -28,6 +30,7 @@ import androidx.compose.runtime.setValue
  * offsets which describe changes to the four edges of a rectangle.
  */
 @Stable
+@Deprecated("accompanist/insets is deprecated")
```

Comment:
Would it be better in cases without a replaceWith to mention something like "it has been superseded by ..." so the user at least knows where to go and look?

Reply:
Done, I updated the deprecated descriptions to briefly explain the equivalents of methods and classes that don't have exact equivalents, and also linked to the migration doc.